bot/exts/info/patreon.py

```diff
@@ -14,9 +14,9 @@
 log = get_logger(__name__)
 
 PATREON_INFORMATION = (
-    "Python Discord is not-for-profit and volunteer run, so we rely on Patreon donations to do what we do. "
-    "We use the money we get to offer excellent prizes for all of our events. Prizes like t-shirts, "
-    "stickers, microcontrollers that support CircuitPython, or maybe even a mechanical keyboard.\n\n"
+    "Python Discord a volunteer run non-profit organization, so we rely on Patreon donations to do what we do. "
+    "We use the money we get to offer excellent prizes for all of our events. These include t-shirts, "
+    "stickers, microcontrollers that support CircuitPython, and maybe even mechanical keyboards.\n\n"
```

Comment:
NIT: "and maybe" --> "and sometimes"

Reply:
I don't think we've ever actually given out a mechanical keyboard, so I removed it from here.